Makefile

```diff
@@ -12,10 +12,10 @@ help:
 ## Initialise local project
 init: .git/.local-hooks-installed
 
-## Clean, build and start local hippo (no autoexport)
+## Clean, build and start local hippo
 serve:
```

Comment:
can you add new target `serve.noexport`?

Reply:
Will do